src/rules/date-rules.ts

```diff
@@ -9,11 +9,24 @@ export const DateRules = {
   }
 }
 
-// validation helper method(s)
+// validation helper methods
 export const DateRuleHelpers = {
+  /** Whether date string to validate is between min date and max date. */
   isBetweenDates (minDate: Date, maxDate: Date, dateStrToValidate: string): boolean {
-    if (!dateStrToValidate) { return true }
+    if (!dateStrToValidate) return true
```

Comment:
A question. Why should it return true in this case?

Reply:
If there is no date string then we cannot compare it below.

Meanwhile, there is other code (on src/components/Registration/StartDate.vue line 88) that validates the date is required.

If you look at the rules array in src/components/Registration/StartDate.vue, the way Vuetify validation works is that it runs each function in the array until a function returns a string instead of True.

Example: the first function checks if the value exists (`!!v`); if False then it returns the error string, otherwise if True then it runs the next function.

So line 16 above should never return True. It is mostly a code safety check.

Reply:
yeah, it makes sense 👍

Reply:
I guess I didn't _really_ answer your question... I think it could just as well return False and the code would work the same way, since THIS function isn't called (via rules array) if the value doesn't exist (first rule would have caught it).

But maybe this validation function is called by other code, where an empty value is valid.

I guess the point is that validating whether the value exists is not THIS function's purpose.